flutter/realm_flutter/android/build.gradle

```diff
@@ -47,3 +47,16 @@ android {
     //     }
     // }
 }
+
+task prePreBuild(type: Exec) {
+    def conf = project.getRootProject().subprojects.find { p -> p.name == 'app' }.android.defaultConfig
+    def targetOsVersion = "API Level ${conf.targetSdkVersion.apiLevel}" // TODO: Map apiLevel to android version name, fx. 18 -> Android 4.3 (API level 18)
```

Comment:
[q] why do we need to map to android version name? I guess the number is sufficient here.

Reply:
Okay, I'll remove the comment

Reply:
My thinking was that OS version for Android is not simply API Level, but obviously API Level is a more precise tag.